modules/data-entry/src/main/frontend/src/questionnaire/AutocreatedQuestion.jsx

```diff
@@ -63,11 +78,31 @@ let AutocreatedQuestion = (props) => {
   // Answer instructions are not displayed since there's nothing the user can do in this form to actually follow them, as the answers are read-only
   return (
     <Question
-      isEdit={false}
-      defaultDisplayFormatter={isFormatted ? (label, idx) => <FormattedText>{label}</FormattedText> : (label, idx) => label}
+      isEdit={isEdit}
+      preventDefaultView
       disableInstructions
-      {...rest}
-    />
+      {...props}
+    >
+      { typeof(existingAnswer?.[1].value) != 'undefined' &&
+        <List sx={{p: 0}}>
+        { Array.of(existingAnswer[1].value).flat().map(v => (
+          <ListItem key={existingAnswer[0]+v} sx={{py: 0}}>
+          { isFormatted ? <FormattedText>{`${v}`}</FormattedText> : v }
```

Comment:
Is there a reason why it's not just:
```suggestion
          { isFormatted ? <FormattedText>{v}</FormattedText> : v }
```

Reply:
FormattedText only accepts children of type string. The questionnaire designer allows you to create an autocreated long question with formatted display. When long value is passed to FormattedText -> WSORD .